pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -736,9 +736,12 @@ func (p *PriorityQueue) addUnschedulableWithoutQueueingHint(logger klog.Logger,
 		// - No unschedulable plugins are associated with this Pod,
 		//   meaning something unusual (a temporal failure on kube-apiserver, etc) happened and this Pod gets moved back to the queue.
 		//   In this case, we should retry scheduling it because this Pod may not be retried until the next flush.
-		p.podBackoffQ.AddOrUpdate(pInfo)
-		logger.V(5).Info("Pod moved to an internal scheduling queue", "pod", klog.KObj(pod), "event", framework.ScheduleAttemptFailure, "queue", backoffQ)
-		metrics.SchedulerQueueIncomingPods.WithLabelValues("backoff", framework.ScheduleAttemptFailure).Inc()
+		queue := p.requeuePodViaQueueingHint(logger, pInfo, queueAfterBackoff, framework.ScheduleAttemptFailure)
```

Comment:
`addUnschedulableWithoutQueueingHint` is called when QueueingHints are disabled, so I maybe `requeuePodViaQueueingHint` should be renamed to something like `requeuePodWithStrategy` to not suggest it's a hint?